From 0b5645042d0d66efc322570872742e7b846a65d0 Mon Sep 17 00:00:00 2001
From: Yehuda Yitschak <yehuday@marvell.com>
Date: Tue, 23 Feb 2016 11:00:46 +0200
Subject: [PATCH 0085/1345] a7040-rz: xor: change xor compatible to z1

commit  e9196bfaf17399d1e5f648f64257a50e7c72853d from
https://github.com/MarvellEmbeddedProcessors/linux-marvell.git

Armada-7040-rz uses AP-806-Z1 which has limitations on the
size of transactions accpetable in the coherency fabric.
Change the xor compatible string to xor z1 to use smaller
transaction sizes

Change-Id: Ic168133f5f6ef5eadf08c3f34095291777380b47
Signed-off-by: Yehuda Yitschak <yehuday@marvell.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm64/boot/dts/marvell/armada-7040-rz.dtsi |   11 +++++++++++
 1 file changed, 11 insertions(+)

diff --git a/arch/arm64/boot/dts/marvell/armada-7040-rz.dtsi b/arch/arm64/boot/dts/marvell/armada-7040-rz.dtsi
index aee462a..a7dfa04 100644
--- a/arch/arm64/boot/dts/marvell/armada-7040-rz.dtsi
+++ b/arch/arm64/boot/dts/marvell/armada-7040-rz.dtsi
@@ -52,4 +52,15 @@
 	model = "Marvell Armada 7040 RZ";
 	compatible = "marvell,armada7040", "marvell,armada-ap806-quad",
 	             "marvell,armada-ap806";
+
+	cpn-110-master {
+		config-space {
+			cpxor0@6a0000 {
+				compatible = "marvell,mv-xor-v2-z1";
+			};
+			cpxor1@6c0000 {
+				compatible = "marvell,mv-xor-v2-z1";
+			};
+		};
+	};
 };
-- 
1.7.9.5

